tests/e2e/external/external_test.go

```diff
@@ -0,0 +1,195 @@
+// This test verifies that two nodes, which can't connect using the local network, are
+// able to still connect using the node-external-ip. In real life, node-external-ip
+// would be a public IP. In the test, we create two networks, one sets the node
+// internal-ip and the other sets the node-external-ip. Traffic is blocked on the former
+
+package validatenodeexternalip
```

Comment:
You have the folder named `external` the test named `External-Ip Suite` the function named `E2EDualStack`.... can you consildate these all to the same name. 

Folder: `externalip`
Package: `externalip`
function: `E2EExternalIP`
suite: `External IP Suite`

Reply:
good idea